src/runtime/d3d12compute.cpp

```diff
@@ -453,7 +460,12 @@ struct d3d12_buffer {
         ReadBack
     } type;
 
-    halide_buffer_t *halide;
+    // UNSAFE to keep a pointer to a 'halide_buffer_t' since it is a POD type
+    // that can be re-assigned / re-scoped outside of this runtime module...
+    //halide_buffer_t *halide;
+    // keeping a copy of it is a bit overkill, so we'll use a flag:
+    bool halide;
```

Comment:
I think there could be a more descriptive name than "halide".

Reply:
definitely needs a better name. I have no idea what this means.

Reply:
Yeap, that's a left-over; `halide` used to be a `halide_buffer_t*` cached within a `d3d12_buffer` for convenience. After I realized it was unsafe to cache such a pointer, I replaced it for a `bool` to minimize the number o changes in the code at that time, but in reality, this field is not necessary at all. It's gone now.